client/src/registerServiceWorker.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
Should we ever re-enable this?

Reply:
I haven't decided what to do with this file yet, might remove it altogether